### ISSUE TITLE
[2019] An Analysis of the Softmax Cross Entropy Loss for Learning-to-Rank with Binary Relevance

### ISSUE THREAD

**Issue by rejasupotaro** [rejasupotaro/paper-reading#72]
## Link

https://ai.google/research/pubs/pub48321

## BibTeX

```
@inproceedings{48321,
title	= {An Analysis of the Softmax Cross Entropy Loss for Learning-to-Rank with Binary Relevance},
author	= {Sebastian Bruch and Xuanhui Wang and Mike Bendersky and Marc Najork},
year	= {2019},
booktitle	= {Proceedings of the 2019 ACM SIGIR International Conference on the Theory of Information Retrieval (ICTIR 2019)}
}
```
